mutornadomon/collectors/ioloop_util.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+
+class UtilizationCollector(object):
+
+    def __init__(self, monitor):
+        self.last_start_time = time.time()
+        self.monitor = monitor
+
+    def __enter__(self):
+        now = time.time()
+        self.last_start_time = now
+
+    def __exit__(self, *args, **kwargs):
+        duration = (time.time() - self.last_start_time)
+        self.last_start_time = time.time()
+        self.monitor.kv('callback_duration', duration)
+        self.monitor.count('accumulated_callback_duration', duration)
```

Comment:
Actually, scratch that. We do need the aggregation, because it can be restarted.
Let's just keep the aggregated time in an attribute, and use `self.monitor.kv` to publish it.

Then, based on that it would be pretty easy to get the utilization metric: `uptime/accumulated_callback_duration`.